docker-bake.hcl

```diff
@@ -128,19 +128,16 @@ target "binary-cross" {
 # dev
 #
 
-variable "DEV_IMAGE" {
```

Comment:
Wondering if this was used anywhere (but could've been in CI previously to tie the image name to PR number 🤔). Guess not really important.